CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 * Support for the s390x platform
   _(The s390x support is currently considered as experimental. We are not aware of any issues, but the s390x build doesn't at this point undergo the same level of testing as the AMD64 container images.)_
 * Update Strimzi Kafka Bridge to 0.21.5
+* Added rebalancing modes on the `KafkaRebalance` custom resource
```

Comment:
I find the names a bit strange. Since it's a type of `KafkaRebalance`, could the first one be simply called `full`? Otherwise should we add `-rebalance` as a prefix to the other two?

Also what about using plural for the scaling modes, ie `add-brokers`, or even using the use cases as names, for example `scale-out`/`scale-in`?

Reply:
I agree on using `full`. Related to the plural, it makes more sense I was just keeping the same name as the endpoints in CC. Not sure about using scale in/out instead of add/remove brokers, I have not a strong opinion. @tomncooper @scholzj wdyt?

Reply:
> Related to the plural, it makes more sense I was just keeping the same name as the endpoints in CC.

+1

> I would prefer sticking with the latter

+1